service/src/main/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderService.java

```diff
@@ -278,7 +278,7 @@ private String formatEPODateTime(LocalDateTime dateTime) {
 
     private String getFormattedRemovalAddress(CaseData caseData) {
         if (caseData.getEpoRemovalAddress() != null) {
-            return caseData.getEpoRemovalAddress().getAddressAsString();
+            return caseData.getEpoRemovalAddress().getAddressAsString(", ");
```

Comment:
Thoughts of replacing this method with an optional, `return Optional.ofNullable(caseData.getEpoRemovalAddress()).map(THIS_LINE).orElse("")`

Reply:
👍